packages/tabs/src/Tabs.tsx

```diff
@@ -184,7 +186,9 @@ function Tabs({
         role="tablist"
         tabIndex={0}
       >
-        {tabs.map((tab, index) => {
+        {tabs?.map((tab, index) => {
+          if (!tab) return null;
```

Comment:
not completely safe for non-TS users since we don't necessarily want to return `null` for values like `0`

Reply:
is this safer: 
```
if (!isComponentType(tab, 'Tab')) {
            return tab
          }
```

Reply:
seems like a reasonable enough behavior to me